.github/workflows/ansible-release.yml

```
@@ -49,10 +47,39 @@ jobs:
       - name: Install dependencies
         working-directory: antsibull
         run: |
+          python3 -m pip install packaging
           python3 -m pip install ansible-core
           python3 -m pip install antsibull
           ansible-galaxy install -r requirements.yml
 
+      - name: Validate version and extract major version
+        shell: python
+        id: extract-version
+        run: |
+          import os
+          import pathlib
+          import sys
+
+          from packaging.version import Version
+
+          FILE_APPEND_MODE = 'a'
+          OUTPUTS_FILE_PATH = pathlib.Path(os.environ['GITHUB_OUTPUT'])
+          VERSION = os.environ['ANSIBLE_VERSION']
+
+          def set_output(name, value):
+              with OUTPUTS_FILE_PATH.open(FILE_APPEND_MODE) as outputs_file:
+                  outputs_file.writelines(f'{name}={value}{os.linesep}')
```

Why not

```suggestion
                  outputs_file.write(f'{name}={value}\n')
```
? You shouldn't need `writelines` to write one line and `\n` does the same thing as `os.linesep` (in text mode, python converts `\n` to `\r\n` if that's `os.linesep` and we're not working with Windows to begin with).

---

This is also copied from my code :)

I have helpers in other places where you can pass an iterable of strings and this is more genetic.